src/sdk/cpp/examples/function_provider/function_provider.cpp

```diff
@@ -22,17 +33,15 @@ class MathFunctions : public FunctionTalent {
     Callee fib;
 
    public:
-    explicit MathFunctions(std::shared_ptr<Publisher> publisher)
-        : FunctionTalent(FEATURE, publisher) {
+    explicit MathFunctions()
+        : FunctionTalent(FEATURE) {
```

Comment:
I think feature is misleading here.. it should be the talent name (e.g. TALENT_NAME as in other files)?

Reply:
I agree, I think we should remove this example and the `event_consumer` example too. These are remnants from our initial experiments. I'll update for now and we can decide whether or not to keep them later.